libdnf-cli/output/repoquery.hpp

```diff
@@ -43,9 +43,8 @@ static struct libscols_table * create_package_info_table(Package & package) {
 
     add_line_into_package_info_table(table, "Name", package.get_name().c_str());
     auto epoch = package.get_epoch();
-    if (epoch != 0) {
-        auto str_epoch = std::to_string(epoch);
-        add_line_into_package_info_table(table, "Epoch", str_epoch.c_str());
+    if (epoch != "0") {
```

Comment:
Do we really want to show epoch as optional value? Can we say that epoch is part of description of the package. When we use list, there is always epoch.

Reply:
```
Name         : dnf
Version      : 4.4.2
Release      : 5g989a0418.fc32
Architecture : src
Size         : 838 k
Source       : None
Repository   : copr:copr.fedorainfracloud.org:rpmsoftwaremanagement:dnf-nightly
Summary      : Package manager
URL          : https://github.com/rpm-software-management/dnf
License      : GPLv2+
Description  : Utility that allows users to manage packages on their systems.
             : It supports RPMs, modules and comps groups & environments.

sudo dnf repoquery dnf 
dnf-0:4.2.19-1.fc32.noarch
dnf-0:4.4.0-3.fc32.noarch
dnf-0:4.4.2-4gf6d1e430.fc32.noarch
dnf-0:4.4.2-4gf6d1e430.fc32.src
dnf-0:4.4.2-5g989a0418.fc32.noarch
dnf-0:4.4.2-5g989a0418.fc32.src

```

Reply:
We should probably just consistently return the epoch.

Reply:
Epoch is now always part of the output.